src/redux/actions/filtered.js

```diff
@@ -0,0 +1,48 @@
+import Lbryio from 'lbryio';
+import * as ACTIONS from 'constants/action_types';
+
+const CHECK_FILTERED_CONTENT_INTERVAL = 60 * 60 * 1000;
+
+export function doFetchFilteredOutpoints() {
+  return dispatch => {
+    dispatch({
+      type: ACTIONS.FETCH_FILTERED_CONTENT_STARTED,
+    });
+
+    const success = ({ outpoints }) => {
+      const splitedOutpoints = [];
+
+      outpoints.forEach((outpoint, index) => {
```

Comment:
instead of creating a new array and pushing to it, you can just use map

```
const formattedOutpoints = outpoints.map(outpoint => {
          const [txid, nout] = outpoint.split(':');

        return { txid, nout: Number.parseInt(nout, 10) };
})
```